pages/auxiliary-sets/[id].js

```diff
@@ -142,8 +131,7 @@ export default function AuxiliarySet({
                   <DataItemValue>{auxiliarySet.moi}</DataItemValue>
                 </>
               )}
-              <DataItemLabel>Summary</DataItemLabel>
-              <DataItemValue>{auxiliarySet.summary}</DataItemValue>
+              <FileSetDataItems item={auxiliarySet} />
```

Comment:
This comment applies to all the FileSet pages. I think you want to wrap the type-specific data items within `FileSetDataItems`. See how `BiosampleDataItems` wraps all the InVitroSystem-specific data items within itself. All the data items it wraps goes into the `children` property of `FileSetDataItems`.

Reply:
done